nolearn/lasagne/visualize.py

```diff
@@ -338,29 +347,31 @@ def make_pydot_graph(layers, output_shape=True, verbose=False):
     return pydot_graph
 
 
-def draw_to_file(layers, filename, **kwargs):
+def draw_to_file(net, filename, **kwargs):
```

Comment:
Let's try to keep backward compatibility and make both layers and net work as an argument.  So maybe keep calling the argument `layers` and further down do: `layers = layers.get_all_layers() if hasattr(layers, 'get_all_layers') else layers`